scripts/ingests/Y_dwarf_pm_ingest.py

```diff
@@ -0,0 +1,59 @@
+#Adapted from Clemence Fontanive's Y-dwarf-astrometry-ingest.py code
+
+#imports from y-dwarf-astrometry-ingest.py, utils.py, ingest_ATLAS_spectral_types.py
+import sys
+sys.path.append('.')
+from astrodbkit2.astrodb import create_database
+from astrodbkit2.astrodb import Database
+from simple.schema import *
+from utils import ingest_pm
+from astropy.table import Table
+import numpy as np
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astroquery.simbad import Simbad
+import warnings
+warnings.filterwarnings("ignore", module='astroquery.simbad')
+import re
+import os
+from pathlib import Path
```

Comment:
Many of these things are not required. In the future, check to be sure you only import what you use.